packages/server/api/src/app/authentication/authentication-service/hooks/community-authentication-hooks.ts

```diff
@@ -31,15 +31,8 @@ export const communityAuthenticationServiceHooks: AuthenticationServiceHooks = {
     let organization = await organizationService.getOldestOrganization();
 
     const isAdminUser = user.email === adminEmail;
-    if (isAdminUser) {
-      if (organization) {
-        throw Error(
```

Comment:
why did you remove this check?

Reply:
I will revert. Originally I allowed creating another admin user, but now the code always updates the existing one.